src/cosmic_ray/commands/init.py

```diff
@@ -2,43 +2,54 @@
 import logging
 from typing import Iterable
 import uuid
-
 from cosmic_ray.ast import get_ast, ast_nodes
 import cosmic_ray.modules
-from cosmic_ray.work_item import MutationSpec, ResolvedMutationSpec, WorkItem
+from cosmic_ray.work_item import ResolvedMutationSpec, WorkItem
 from cosmic_ray.plugins import get_operator
 from cosmic_ray.work_db import WorkDB
 
 log = logging.getLogger()
 
 
-def _all_work_items(module_paths, operator_names) -> Iterable[WorkItem]:
+def _all_work_items(module_paths, operator_cfgs) -> Iterable[WorkItem]:
     "Iterable of all WorkItems for the given inputs."
+
     for module_path in module_paths:
         module_ast = get_ast(module_path)
 
-        for op_name in operator_names:
-            operator = get_operator(op_name)()
+        for operator_cfg in operator_cfgs:
+            operator_name = operator_cfg["name"]
+            if "args" not in operator_cfg:
+                operator_args = [{}]
+            else:
+                operator_args = operator_cfg["args"]
 
-            positions = (
-                (start_pos, end_pos)
-                for node in ast_nodes(module_ast)
-                for start_pos, end_pos in operator.mutation_positions(node)
-            )
+            for args in operator_args:
+                try:
```

Comment:
This concerns me a bit. If there's a typo in an argument name in the config, we'll silently ignore that here by assuming the resulting TypeError means that the operator didn't get *any* args. A typo like this should result in an error, I think.

I'd be more comfortable doing something like this. If there's a TypeError and `args` is empty, we skip the operator. If `args` is *not* empty, though, we should raise an exception and halt initialization. This way we can let the user know that they've made what appears to be a mistake.

Reply:
Good point. I hadn't thought of this corner case.

Your solution makes sense; I'll implement the changes soon.